src/generate_list.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+################################################################################
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2017 Massachusetts Institute of Technology.
+################################################################################
+
+# Configure the installer here
+INITRAMFS_TOOLS_GIT=https://salsa.debian.org/kernel-team/initramfs-tools.git
```

Comment:
Hmm yeah we should probably ship a tool in cosa for this, or patch dracut.  Last time I looked at this concatenated initramfs images were messy and I think even initramfs-tools didn't handle some of them.